### PR TITLE
fix: Embed/Show header in weekly view and column view

### LINKED ISSUE #10165: [CAL-2173] /embed: weekly view - header is broken and I can't move to next week
The position of the fixed day header is broken in embeds and the top header showing the date and allowing me to move forward in the month is missing.

[https://github.com/calcom/cal.com/assets/4536123/fd7d5bca-e73e-453d-93c7-8388b20ccc94](https://github.com/calcom/cal.com/assets/4536123/fd7d5bca-e73e-453d-93c7-8388b20ccc94)

Steps to reproduce:

* Embed an event type in a website using the "floating pop-up button" option
* Click the "book a call" button when on the website.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2173](https://linear.app/calcom/issue/CAL-2173/embed-weekly-view-header-is-broken-and-i-cant-move-to-next-week)</sub>

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

[After Fix](https://www.loom.com/share/4b3d45294ccc47baa0163b32222fcdbd)
Fixes #10165
## Type of change
  - [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
- See the loom

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist
- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/features/bookings/Booker/components/Header.tsx [hunk 1]
```diff
@@ -43,7 +42,7 @@ export function Header({
     [setLayout, layout]
   );
 
-  if (isMobile || isEmbed || !enabledLayouts) return null;
+  if (isMobile || !enabledLayouts) return null;
```
Comment (hariombalhara) [inline packages/features/bookings/Booker/components/Header.tsx:45]: We don't want to hide the entire header in embed. We just want the layout toggle to go away for embed.

File: packages/features/bookings/Booker/components/Header.tsx [hunk 2]
```diff
@@ -119,7 +118,12 @@ const LayoutToggle = ({
   layout: string;
   enabledLayouts?: BookerLayouts[];
 }) => {
+  const isEmbed = typeof window !== "undefined" && window?.isEmbed?.();
+
   const { t } = useLocale();
+  if (isEmbed) {
```
Comment (hariombalhara) [inline packages/features/bookings/Booker/components/Header.tsx:124]: Don't show layout toggle in embed.

**Review by sean-brydon — APPROVED** [review]
Havent tested but good looks safe n good to merge :)

**Comment by Jaibles** [comment]
Thanks @hariombalhara 🙏

**Pull request merged**